### ISSUE TITLE
Error on RequestHandlers/Sequential.js while using snekfetch

### ISSUE THREAD

**Issue by Wodki** [hydrabolt/discord.js#1903]
**Please describe the problem you are having in as much detail as possible:**
So I am using the discordjs lib quite long time and have never had this issue before so it perhaps has something to do with the recent update. The error is below.

```
TypeError: Cannot read property 'options' of undefined
at item.request.gen.end (../node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:69:74)
at then (../node_modules/snekfetch/src/index.js:263:21)
at process._tickDomainCallback (internal/process/next_tick.js:129:7)
```

I had never had this issue before but as of two days ago it has happened four-five times at competely random intervals.

**Include a reproducible code sample here, if possible:**
Unfortunately I can't find where it occurs so I can't provide any code. It has occured in many commands that interact with users so I'll go ahead and say that its not in a specific part of the code.

**Further details:**

- discord.js version: 11.2.1
- node.js version: 8.4
- Operating system: centOS
- Priority this issue should have – please be realistic and elaborate if possible: high

**Comment by busheezy** [comment]
I'm not sure where or why but I'm experiencing the same thing.

**Comment by devsnek** [comment]
this change isn't in master, gimme a sec

**Comment by busheezy** [comment]
I very much appreciate the quick reply.

**Comment by busheezy** [comment]
Sorry to bother. Is this included in the newest release?

**Comment by SpaceEEC** [comment]
Yes, the referenced PR (#1904) is included in the newest release (11.3).

**Comment by busheezy** [comment]
Thank you! I looked for it but not well enough.
